features/gezag-minderjarige/bevoegdheid-tot-gezag.feature

```diff
@@ -0,0 +1,112 @@
+# language: nl
+
+Functionaliteit: Gezag bepalen voor een minderjarige wanneer een of beide ouders niet bevoegd zijn tot gezag
+  Een persoon is onbevoegd tot gezag wanneer die:
+  - is overleden
+  - onder curatele staat
+  - is minderjarig
+
+
+  Achtergrond:
+    Gegeven de persoon 'Gerda' met burgerservicenummer '000000012'
+    * is meerderjarig
+    En de persoon 'Aart' met burgerservicenummer '000000024'
+    * is meerderjarig
+    En de persoon 'Ariana' met burgerservicenummer '000000036'
+    * is meerderjarig
+    En de persoon 'Bert' met burgerservicenummer '000000048'
+    * is minderjarig
+
+
+  Regel: Er is sprake van eenhoofdig ouderlijk gezag als één van de ouders met tweehoofdig ouderlijk gezag overleden of niet bevoegd is
```

Comment:
waarom wordt het overleden zijn van een ouder expliciet benoemd? Overleden zijn valt ook onder niet bevoegd als je naar de definitie van onbevoegd bekijkt op regel 4 t/m 7

Reply:
In de documentatie die we hebben overgenomen en in de implementatie maakt overleden deel uit van (on)bevoegdheid. Daar zijn we het niet mee eens, waardoor overleden uit (on)bevoegdheid is gehaald. Hierdoor spreken we voortaan van "overleden of niet bevoegd".

Reply:
We passen de feature titel aan naar "Gezag bepalen voor een minderjarige wanneer een of beide ouders zijn overleden of niet bevoegd zijn tot gezag"
Met daarbij de toelichting:
>  Een persoon is onbevoegd tot gezag wanneer die:
>  - onder curatele staat
>  - is minderjarig